Content.Server/Power/EntitySystems/BatterySystem.cs

```diff
@@ -7,11 +8,14 @@ namespace Content.Server.Power.EntitySystems
     [UsedImplicitly]
     public sealed class BatterySystem : EntitySystem
     {
+        private const float PricePerJoule = 0.0001f;
```

Comment:
Everything looks good, it might be worth considering making this a field on battery that defaults to 0.0001f. Maybe certain items are more/less convenient to get the energy out of and this effects price

Reply:
I did it, and it allows better pricing of the power-cells